lib/dpl/provider/cargo.rb

```diff
@@ -0,0 +1,22 @@
+require 'json'
+require 'uri'
+
+module DPL
+  class Provider
+    class Cargo < Provider
+      def needs_key?
+        false
+      end
+
+      def check_auth
+        option(:token)
```

Comment:
Is there a way to check with `cargo` if the provided token is valid? If so, it is best to do it here, so that the problem is detected early.

Reply:
Not currently, I might open an issue and see about adding a command for it to `cargo`.